moduleframework/common.py

```diff
@@ -39,8 +38,35 @@
 import requests
 import warnings
 import ast
+from glob import glob
 from avocado.utils import process
-from moduleframework.mtfexceptions import ModuleFrameworkException, ConfigExc, CmdExc, DefaultConfigExc
+from mtfexceptions import ModuleFrameworkException, ConfigExc, CmdExc, DefaultConfigExc
+from core import DATADIR, is_debug, is_not_silent, print_info, print_debug
```

Comment:
Why not only `import core`. print_info is so general and we don't know from which module comes from.